Annotation/declarative_golden_set.json

```diff
@@ -1371,7 +1469,7 @@
                     }
                 ],
                 "type": "declarative",
-                "clause": "",
+                "clause": "that the FBI\u2019s confidential informant was material to his defense",
                 "embedded clauses": []

```

Comment:
wrong clause, so "that Abdulkair failed to show that xx" and then the nested EC "that the FBI xx"